### ISSUE TITLE
not able to pass params to $http.get when running on server.

### ISSUE THREAD

**Issue by haukurk** [netanelgilad/angular-meteor-server#4]
I've been testing out to make server side $http calls with AngularJS, and so far it has not been working out.

I have these following two services, one client side and one server side:

Service in server folder:

```
angular.module('testApp')
  .service('TestServerService', function ($http) {
    var service = {};

    // public members
    function SearchTranslations(searchParams) {
        return $http.get("http://testapi.hauxi.is/api/testmethod", { params: searchParams })
                    .then(handleSuccess, handleError('Error when retreiving data from api.'));
    }

    // private members
    ...
    ...

    service.SearchTranslations = SearchTranslations;
    return service;
  })
  .config(function (ServerAPIProvider) { // Expose it to the API Provider.
    ServerAPIProvider.register('TranslationService');
  });
```

Service in client folder:

```
angular.module('testApp')
  .service('TestClientService', function ($http) {
    var service = {};

    // public members
    function SearchTranslations(searchParams) {
        return $http.get("http://testapi.hauxi.is/api/testmethod", { params: searchParams })
                    .then(handleSuccess, handleError('Error when retreiving data from api.'));
    }

    // private members
    .... 
    ....

    service.SearchTranslations = SearchTranslations;
    return service;
  });
```

The client service works perfectly like always, but the server service discards the params passed to the function.

In my case searchParams can be like:

```
httpParams = {
   "filter[limit]": 100,
   "filter[where][word][like]": "searching%"
}
```

When using the client side service I see these logs in on the API server:
[21/Jul/2015:19:00:00 -0400] "GET /api/translations?filter[where][word[like]=hauku%&filter[limit]=100 HTTP/1.1" 200 153 "-" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.134 Safari/537.36"

When using the server side service I see these logs on the API server:
[21/Jul/2015:20:27:58 -0400] "GET /api/translations HTTP/1.1" 200 15418540 "-" "-"

Note, that no params are used when using the server service..

Any idea why this is happening?

**Comment by netanelgilad** [comment]
@haukurk Oh I now see that you moved the issue from the other repo over here.
Then I'll take a look at this probably tomorrow, adding support for http params should not be difficult.

**Comment by jamalx31** [comment]
I have the same problem +1

**Comment by netanelgilad** [comment]
Fixed in 1.4.5
